Sample.java

```diff
@@ -1,7 +1,53 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Time Complexity : 
+  //add: O(n) because tmp.contains will iterate throught the entire list
+  //remove: O(1)
+  //contains: O(n) because tmp.contains will iterate throught the entire list
+  
```

Comment:
Please try to refactor your code to have O(1) in both add and remove methods. This is the key feature of hash Set to have O(1) time complexity.